<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(196, 232, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="motor_for_click">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>电机正转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motor_back_click">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>电机反转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motor_stop">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="text_log">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_17">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcd_temp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_18">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcd_humi">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_19">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcd_lig">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>亮度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_serial">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>系统开关</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcd_dis">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 226, 107);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>距离</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_smok">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>烟雾报警器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>（0表示安全状态）</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="ring">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>红外报警</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>开门</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>身份识别</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="id_judge">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>111</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>老人居家系统</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
